.gitattributes

```diff
@@ -0,0 +1,8 @@
+/tests export-ignore
```

Comment:
@kachkaev Symfony explicitly reverted the exclusion of tests of the archive a while ago